src/operator/tensor/elemwise_binary_broadcast_op.h

```diff
@@ -267,8 +267,6 @@ void BinaryBroadcastBackwardUseIn(const nnvm::NodeAttrs& attrs,
   }
 }
 
-#undef BROADCAST_NDIM_SWITCH
```

Comment:
This line prevents me from using this macro in broadcast_reduce_op.h. So I had to remove it.

I will remove the function definitions of ReduceToAssign.